profile/README.md

```diff
@@ -12,3 +12,4 @@ Or get the Seam SDK in your favorite technology:
 * [React](https://github.com/seamapi/react)
 * [Ruby](https://github.com/seamapi/ruby)
 * [PHP](https://github.com/seamapi/php)
+* [C#](https://github.com/seamapi/csharp)
```

Comment:
maybe the order needs to be as suggested in another PR
https://github.com/seamapi/.github/pull/2/files#r1889298423